src/DatePicker/DatePicker.js

```diff
@@ -220,14 +222,18 @@ DatePicker.propTypes = {
     enableRangeSelection: PropTypes.bool,
     inputProps: PropTypes.object,
     locale: PropTypes.string,
+    popoverStyle: PropTypes.object,
     onBlur: PropTypes.func
 };
 
 DatePicker.defaultProps = {
     buttonLabel: 'Choose date',
     defaultValue: '',
     locale: 'en',
-    onBlur: () => {}
+    onBlur: () => {},
+    popoverStyle: {
+        width: '100%'
```

Comment:
1. Can you show the issue (screenshot) that was present before putting this style in? 
2. Does it make sense to set a default style in here rather than making a change in `fundamental-styles`? And once a consumer sets this prop, they would need to either include the `width` change or it would be overwritten.

Reply:
Is this trying to solve the Popover width sizing issues? We can set a default of one of these options instead: https://fundamental-react.netlify.com/popover#_width-_sizing-_type

Reply:
<img width="714" alt="input width not matching that of parent" src="https://user-images.githubusercontent.com/3882561/75307326-6e714380-5886-11ea-9d21-1453aeed1009.png">

Hi as you can see above, each input field is given a width of 240px, the ```SearchInput``` and ```DatePicker```'s width would not match that of their parents.

Reply:
Did `popperProps={{ widthSizingType: "matchTarget" }}` not work for this use case?

Reply:
<img width="1569" alt="input width not matching that of parent even with widthSizingType as 'matchTarget'" src="https://user-images.githubusercontent.com/3882561/75409303-47347800-5953-11ea-9859-f72f5945d4b5.png">

As you can see from above, passing ```widthSizingType={"matchTarget"}``` actually did not fix this issue.

Reply:
@jbadan There's a mixup between the "control" element (InputGroup) and the popper content itself. The issue here is the DatePicker InputGroup isn't aligning widths with the rest of the form fields in the screenshot. 

The `popoverStyle` prop here only affects the styling on the main popover div (with the control element), whereas `widthSizingType` is for the popper content itself and wouldn't affect it.

![image](https://user-images.githubusercontent.com/22511993/75483104-f88adc00-595a-11ea-9f31-ff27835413b2.png)

The naming of the prop is a little misleading.

Reply:
Ah ok - sorry for the misunderstanding. This looks like a change needs to happen in fundamental-styles https://github.com/SAP/fundamental-styles/blob/master/src/popover.scss#L49